Cargo.toml

```diff
@@ -40,9 +40,9 @@ features = [
 ]
 
 [target."cfg(target_os = \"linux\")".dependencies]
-gtk = "0.16"
-gdk = "0.16"
-gdk-pixbuf = "0.16"
+gtk = "0.18"
+gdk = "0.18"
+gdk-pixbuf = "0.18"
```

Comment:
We should probably remove these and just use the re-exported modules in `gtk` crate

Reply:
Makes sense, I'll do some refactoring.